frontend/public/components/monitoring/receiver-forms/pagerduty-receiver-form.tsx

```diff
@@ -21,218 +31,202 @@ const GLOBAL_FIELDS = [
+          label={t('public~Prometheus')}
+          value="prometheus"
+          onChange={(e) =>
+            dispatchFormChange({
+              type: 'setFormValues',
+              payload: { pagerdutyIntegrationKeyType: (e.target as HTMLInputElement).value },
+            })
+          }
+          isChecked={formValues.pagerdutyIntegrationKeyType === 'prometheus'}
+          data-checked-state={formValues.pagerdutyIntegrationKeyType === 'prometheus'}
+          data-test-id="integration-type-prometheus"
+        />
+      </FormGroup>
+      <FormGroup
+        label={
+          formValues.pagerdutyIntegrationKeyType === 'events'
             ? t('public~Routing key')
-            : t('public~Service key')}
-        </label>
-        <span className="pf-v6-c-form-control">
-          <input
-            type="text"
-            aria-describedby="integration-key-help"
-            id="integration-key"
-            data-test-id="integration-key"
-            value={formValues.pagerdutyIntegrationKey}
-            onChange={(e) =>
-              dispatchFormChange({
-                type: 'setFormValues',
-                payload: { pagerdutyIntegrationKey: e.target.value },
-              })
-            }
+            : t('public~Service key')
+        }
+        fieldId="integration-key"
+        isRequired
+      >
+        <TextInput
+          type="text"
+          id="integration-key"
+          data-test="integration-key"
+          value={formValues.pagerdutyIntegrationKey}
+          onChange={(_e, value: string) =>
+            dispatchFormChange({
+              type: 'setFormValues',
+              payload: { pagerdutyIntegrationKey: value },
+            })
+          }
+        />
+        <FormHelperText>
+          <HelperText>
+            <HelperTextItem>{t('public~PagerDuty integration key.')}</HelperTextItem>
+          </HelperText>
+        </FormHelperText>
+      </FormGroup>
+      <Grid hasGutter>
+        <GridItem span={7}>
+          <FormGroup label={t('public~PagerDuty URL')} fieldId="pagerduty-url" isRequired>
+            <TextInput
+              type="text"
+              id="pagerduty-url"
+              data-test="pagerduty-url"
+              value={formValues.pagerduty_url}
+              onChange={(_e, value: string) =>
+                dispatchFormChange({
+                  type: 'setFormValues',
+                  payload: { pagerduty_url: value },
+                })
+              }
+            />
+            <FormHelperText>
+              <HelperText>
+                <HelperTextItem>
+                  {t('public~The URL of your PagerDuty installation.')}
+                </HelperTextItem>
+              </HelperText>
+            </FormHelperText>
+          </FormGroup>
+        </GridItem>
+        <GridItem span={1} /> {/* fixes an overlapping control issue */}
+        <GridItem span={4}>
+          <SaveAsDefaultCheckbox
+            formField="pagerdutySaveAsDefault"
```

Comment:
Aligns `checkbox` with 'input'
```suggestion
            className="pf-v6-u-align-content-center"
            formField="pagerdutySaveAsDefault"
```